### PR TITLE
Update 01-sql-basic-queries.md

### PR DESCRIPTION
**Pull request by buehlerd**
Fixed a typo and added a comment (mainly to test using Github, but hopefully useful too).

### CODE REVIEW

**Comment by buehlerd** [comment]
Fixed a typo and added a comment (mainly to test using Github, but hopefully useful too).

File: _episodes/01-sql-basic-queries.md [hunk 1]
```diff
@@ -44,9 +44,7 @@ Or we can select all of the columns in a table using the wildcard *
 
 ### Limiting results
 
-Sometimes you don't want to see all the results you just want to get a sense of
-of what's being returned. In that case you can use the LIMIT command. In particular
-you would want to do this if you were working with large databases.
+Sometimes you don't want to see all the results you just want to get a sense of what's being returned. In that case, you can use the LIMIT command. In particular, you would want to do this if you were working with large databases.
```
Comment (tpoisot) [inline _episodes/01-sql-basic-queries.md]: "want to see all the results you just" -> "want to see all the results, you just"

Comment (tpoisot) [reply _episodes/01-sql-basic-queries.md]: "you can use the LIMIT command" -> "you can use the `LIMIT` command"

Comment (ChristinaLK) [inline _episodes/01-sql-basic-queries.md:47]:
```suggestion
Sometimes you don't want to see all the results, you just want to get a sense of what's being returned. In that case, you can use the `LIMIT` command. In particular, you would want to do this if you were working with large databases.
```

**Pull request merged**